src/logger/fileWriter.c

```diff
@@ -149,13 +152,13 @@ static int write_samples_data(const LoggerMessage *msg)
         size_t count = msg->sample->channel_count;
 
         if (NULL == sample) {
-                pr_warning("Logger: null sample record\r\n");
+                pr_warning("_RCP_BASE_FILE_: null sample record\r\n");
```

Comment:
https://trello.com/c/f5SizLe0/308-establish-logging-prefix-standard-for-firmware-and-go-with-it tracks doing this.